server/dash/dashboards/overviewComb.py

```diff
@@ -0,0 +1,199 @@
+nc_req_count_api_params = {'field':'council_name', 'filter':'created_date>=2016-01-01'}
+NC_REQ_COUNT_DATA_API_PATH = REPORT_API_PATH_ROOT + urllib.parse.urlencode(nc_req_count_api_params)
+print(" * Downloading data from API path: " + NC_REQ_COUNT_DATA_API_PATH)
+nc_req_count_df = pd.read_json(API_HOST + NC_REQ_COUNT_DATA_API_PATH)
+nc_req_count_df = nc_req_count_df.groupby(['council_name'])['counts'].sum().sort_values().to_frame()
+print(" * Dataframe has been loaded from API path: " + NC_REQ_COUNT_DATA_API_PATH)
+
+# Loading the data for the number of new requests
+new_req_count_api_params = {'field':'created_year', 'filter':'created_date>=2016-01-01'}
+NEW_REQ_COUNT_DATA_API_PATH = REPORT_API_PATH_ROOT + urllib.parse.urlencode(new_req_count_api_params)
+print(" * Downloading data from API path: " + NEW_REQ_COUNT_DATA_API_PATH)
+new_req_count_df = pd.read_json(API_HOST + NEW_REQ_COUNT_DATA_API_PATH)
+new_req_count_df = new_req_count_df.groupby(['created_year'])['counts'].sum().to_frame()
+print(" * Dataframe has been loaded from API path: " + NEW_REQ_COUNT_DATA_API_PATH)
+
+# Loading the count of each request types overall
+req_count_api_params = {'field':'type_name', 'filter':'created_date>=2016-01-01'}
+REQ_COUNT_DATA_API_PATH = REPORT_API_PATH_ROOT + urllib.parse.urlencode(req_count_api_params)
+print(" * Downloading data from API path: " + REQ_COUNT_DATA_API_PATH)
+req_count_df = pd.read_json(API_HOST + REQ_COUNT_DATA_API_PATH)
+req_count = req_count_df.groupby(['type_name'], as_index=False)['counts'].sum()
+print(" * Dataframe has been loaded from API path: " + REQ_COUNT_DATA_API_PATH)
+
+# Loading the total number of request source
+req_source_count_api_params = {'field':'source_name', 'filter':'created_date>=2016-01-01'}
+REQ_SOURCE_COUNT_DATA_API_PATH = REPORT_API_PATH_ROOT + urllib.parse.urlencode(req_source_count_api_params)
+print(" * Downloading data from API path: " + REQ_SOURCE_COUNT_DATA_API_PATH)
+req_source_count_df = pd.read_json(API_HOST + REQ_SOURCE_COUNT_DATA_API_PATH)
+req_source_count = req_source_count_df.groupby(['source_name'])['counts'].sum()
+print(" * Dataframe has been loaded from API path: " + REQ_SOURCE_COUNT_DATA_API_PATH)
+
+# Loading the number of Request Agencies
+req_agency_count_api_params = {'field':'agency_name', 'filter':'created_date>=2016-01-01'}
+REQ_AGENCY_COUNT_DATA_API_PATH = REPORT_API_PATH_ROOT + urllib.parse.urlencode(req_agency_count_api_params)
+print(" * Downloading data from API path: " + REQ_AGENCY_COUNT_DATA_API_PATH)
+req_agency_count_df = pd.read_json(API_HOST + REQ_AGENCY_COUNT_DATA_API_PATH)
+agency_count = req_agency_count_df.groupby(['agency_name'])['counts'].sum()
+print(" * Dataframe has been loaded from API path: " + REQ_AGENCY_COUNT_DATA_API_PATH)
+
+# Request Share by Agency Pie Chart
+req_agency_count_df = agency_count.to_frame()
+req_agency_count_df.sort_values('counts', ascending=False, inplace=True)
+req_agency_count_df.loc['Others'] = req_agency_count_df[4:].sum()
```

Comment:
how was 4 selected? Please make a constant for it.

Reply:
Thanks for the suggestion - resolved